### DISCUSSION TITLE
DockerImage Tag

### DISCUSSION THREAD

**Discussion by venkatsai786** [changesets/changesets#1080] [Q&A]
Can i update the Docker Image number using changesets

**Comment by Andarist** [comment]
You would have to write an additional script. For instance, I have a VS Code extension with `package.json#private: true` in it. It participates in version bumping and changelog generation as usual. But to publish it, we have to write an extra step in the release command (see [here](https://github.com/statelyai/xstate-tools/blob/4be03b040cb67d614517e77e84285aeed6a022b1/package.json#L33)). Note though that this always tries to publish this extension (when the publish happens) and it might fail because the current version is already published - but this script is just "swallowing" this error so the whole thing doesn't actually fail in such a situation.
